muon/transport/amqp/server.js

```diff
@@ -54,7 +54,7 @@ function initMuonClientServerSocket(amqpApi, listen_queue, send_queue, serverSta
 
      serverStackChannel.listen(function(event) {
             logger.debug("[*** TRANSPORT:SERVER:OUTBOUND ***]  handling outbound muon event to queue " + send_queue + ": %s", JSON.stringify(event));
-            amqpApi.outbound(send_queue).send({headers: {}, data: event});
+            amqpApi.outbound(send_queue).send({headers: {"content_type":"application/json"}, data: event});
```

Comment:
content_type is a required field in the schema. not having it there breaks the java side as it assumes it